src/System.Configuration.ConfigurationManager/tests/System/Configuration/UrlPathTests.cs

```diff
@@ -0,0 +1,157 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Configuration;
+using System.IO;
+using Xunit;
+
+namespace System.ConfigurationTests
+{
+    public class UrlPathTests
+    {
+        private const string TEST_FILE_NAME = "TestFileForUrlPathTests.txt";
```

Comment:
Hi @dhoehna thanks for your first contribution.
Please check ou the style guidelines https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md which require PascalCasing.